src/features/prefix.json

```diff
@@ -0,0 +1,8 @@
+{
+  "prefixes": [
```

Comment:
this file seems a bit strange to me.
Why do you need a json format?

Reply:
No need.

Will change!